pkg/model/components/etcdmanager/model.go

```diff
@@ -212,6 +215,8 @@ spec:
       path: /etc/kubernetes/pki/etcd-manager
       type: DirectoryOrCreate
     name: pki
+  - name: bin
+    emptyDir: {}
```

Comment:
We should double check what `medium` is used here - as we don't normally run with swap, I think we would prefer this not be in RAM

Reply:
By default, emptyDir volumes are mounted using local ephemeral storage.
https://kubernetes.io/docs/concepts/storage/volumes/#emptydir
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage